modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/template/mgt/TemplateManagementTestCase.java

```diff
@@ -0,0 +1,70 @@
+package org.wso2.identity.integration.test.template.mgt;
```

Comment:
License header is missing

Reply:
fixed with https://github.com/wso2/product-is/pull/9220.